### ISSUE TITLE
Sample vs Frame

### ISSUE THREAD

**Issue by cconcolato** [Netflix/vp9-dash#6]
Can you clarify what the following sentence mean ?

> Video sample storage in the generic binding uses a simple mapping to frames defined in the codec
> specification.
> Does it mean the sample data corresponds to the `frame()` syntactical element from the spec?

**Comment by dronca** [comment]
Yes.  1 sample is one frame as defined in 6.1 of the VP9 specification.  I will revise the working to remove this ambiguity.
